Home.html

```diff
@@ -14,64 +14,98 @@
-            </div>
-                <div class="main-program-divs d-flex ps-3 pe-3 ms-3 me-3 mb-2">
-                        <img src="assets/visa.png" alt="Visa">
-                        <h6>Visa</h6>
-                        <p>Visa application services for more than 10 countries</p>
-                </div>
-                <div class="main-program-divs d-flex ps-3 pe-3 ms-3 me-3 mb-2">
-                    <img src="assets/tickets.png" alt="Tickets">
-                    <h6>Tickets</h6>
-                    <p>We take care of the tickets so you don't have to worry about a thing</p>
-                </div>
-                <div class="main-program-divs d-flex ps-3 pe-3 ms-3 me-3 mb-2">
-                    <img src="assets/bank.png" alt="Bank">
-                    <h6>Bank Statement</h6>
-                    <p>Don't have a bank statement, no problem we will take care of that</p>
-                </div>
-                <div class="main-program-divs d-flex ps-3 pe-3 ms-3 me-3 mb-2">
-                    <img src="assets/patient.png" alt="passport">
-                    <h6>Medical Visa</h6>
-                    <p>Including all the medical files you need to apply for a medical visa</p>
+                <div class="program-container">
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                        <img src="assets/passport.png" alt="passport">
+                        <h6>passport</h6>
+                        <p>Online form application services for getting your passport in no time</p>
+                    </div>
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                            <img src="assets/visa.png" alt="Visa">
+                            <h6>Visa</h6>
+                            <p>Visa application services for more than 10 countries</p>
+                    </div>
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                        <img src="assets/tickets.png" alt="Tickets">
+                        <h6>Tickets</h6>
+                        <p>We take care of the tickets so you don't have to worry about a thing</p>
+                    </div>
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                        <img src="assets/bank.png" alt="Bank">
+                        <h6>Bank Statement</h6>
+                        <p>Don't have a bank statement, no problem we will take care of that</p>
+                    </div>
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                        <img src="assets/patient.png" alt="passport">
+                        <h6>Medical Visa</h6>
+                        <p>Including all the medical files you need to apply for a medical visa</p>
+                    </div>
```

Comment:
- [ ] I think it would be better if you fix the blow issue where the content of the `Main Program` section overlaps on the header while scrolling
![Screenshot from 2023-02-11 00-08-04](https://user-images.githubusercontent.com/83042742/218208499-ea536d67-d581-492f-9ae1-dbe405bb3dd0.png)